src/webcomponents/file/file-browser.js

```diff
@@ -108,6 +107,16 @@ export default class OpencgaFileBrowser extends LitElement {
         }
     }
 
+    render() {
+        return this.opencgaSession && this._config ? html`
```

Comment:
Please instead of doing this use an `if` stament:

```
if (!this.opencgaSession || !this._config) {
    return html`...`;
}

return html`
    ...
`;
```